### ISSUE TITLE
Adding Logistic Regression with enet

### ISSUE THREAD

**Issue by jreps** [OHDSI/PatientLevelPrediction#265]
The PatientLevelPrediction package updates make it easy to add R classifies into the package.  It would be useful to add Logistic Regression with enet into the package.

**Comment by ted9219** [comment]
Elastic net (which combines the ridge and lasso) will be a great option for PLP.

**Comment by egillax** [comment]
I don't think we want to add ENet, which is in R I believe? It's probably far to slow. Glmnet could work but as current research shows: https://academic.oup.com/jamia/article/31/7/1514/7676584
ElasticNet provides bigger models (i.e. complexity) at same performance.
